packages/block-editor/src/components/block-list/index.js

```diff
@@ -230,10 +263,34 @@ function Items( {
 						! selectedBlocks.includes( clientId )
 					}
 				>
+					{ isSectionBlock( clientId, sectionClientIds ) && (
+						<div
+							className={ clsx( 'zoom-out-separator', {
+								'is-open':
```

Comment:
We could use this same condition to load the div so we don't have a bunch of empty divs in the code, but then we need to figure out a different way to animate them